deployment/centos-package-x64/Dockerfile

```diff
@@ -8,22 +8,29 @@ ARG SDK_VERSION=2.2
 ENV SOURCE_DIR=/jellyfin
 ENV ARTIFACT_DIR=/dist
 
-# Prepare CentOS build environment
+# Prepare CentOS environment
 RUN yum update -y \
- && yum install -y @buildsys-build rpmdevtools yum-plugins-core libcurl-devel fontconfig-devel freetype-devel openssl-devel glibc-devel libicu-devel nodejs \
- && rpm -Uvh https://packages.microsoft.com/config/rhel/7/packages-microsoft-prod.rpm \
+ && yum install -y epel-release
+
+# Install build dependencies
+RUN yum install -y @buildsys-build rpmdevtools yum-plugins-core libcurl-devel fontconfig-devel freetype-devel openssl-devel glibc-devel libicu-devel nodejs wget git
```

Comment:
any reason for this being a separate `RUN`? this seems to be creating an extra layer which isn't needed

Reply:
For my purposes, it avoided redoing the update + epel release step (about 15 minutes) every time a new dependency was added. Since we're not publishing these containers, I'm not concerned about additional layers and would prefer these commands to be discrete steps.